ui/src/app/archived-workflows/components/archived-workflow-details/archived-workflow-details.tsx

```diff
@@ -280,6 +270,36 @@ export class ArchivedWorkflowDetails extends BasePage<RouteComponentProps<any>,
             });
     }
 
+    private resubmitArchivedWorkflow() {
+        if (!confirm('Are you sure you want to resubmit this archived workflow?')) {
```

Comment:
minor -  elsewhere in the UI we use a different confirm dialogue, might be good to use that for consistency